src/epd2in9b/graphics.rs

```diff
@@ -0,0 +1,45 @@
+use crate::epd2in9b::{DEFAULT_BACKGROUND_COLOR, HEIGHT, NUM_DISPLAY_BITS, WIDTH};
+use crate::graphics::{Display, DisplayRotation};
+use crate::prelude::*;
+use embedded_graphics::prelude::*;
+
+pub struct Display2in9b {
+    buffer: [u8; NUM_DISPLAY_BITS as usize],
+    rotation: DisplayRotation,
+}
+
+impl Default for Display2in9b {
+    fn default() -> Self {
+        Display2in9b {
+            buffer: [DEFAULT_BACKGROUND_COLOR.get_byte_value(); NUM_DISPLAY_BITS as usize],
+            rotation: DisplayRotation::default(),
+        }
+    }
+}
+
+impl Drawing<Color> for Display2in9b {
```

Comment:
I merged my updated branch yesterday, so you need to update this to the newest emedded-graphics version too.